### PR TITLE
12.0 mig partner address version

### PR DESCRIPTION
**Pull request by kevinkhao**

### CODE REVIEW

**Comment by kevinkhao** [comment]
Note the related PR https://github.com/OCA/partner-contact/pull/888

File: partner_address_version/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Copyright 2018 Akretion - Benoît Guillot
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_address_version/__init__.py:2]: Please change all license lines to `https:`

File: partner_address_version/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (NL66278) [inline partner_address_version/models/__init__.py:1]: There should be no copyright lines in init files.

File: partner_address_version/tests/test_address_version.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+from odoo.exceptions import UserError
+
+
+class TestAddressVersion(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestAddressVersion, cls).setUpClass()
+        cls.partner_vals = OrderedDict(
+            [
+                ("name", u"Name"),
```
Comment (NL66278) [inline partner_address_version/tests/test_address_version.py:16]: You do not need the `u` Unicode specifier on this. All strings are Unicode in python3.

**Review by NL66278 — APPROVED** [review]
:+1: Nothing blocking, but would be nice if you could attend to comments.

**Pull request merged**